misc/nsis/install.nsh

```diff
@@ -435,29 +435,18 @@ Function .onInit
   ${If} ${RunningX64}
     ; https://learn.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-osversioninfoa#remarks
     GetWinVer $R0 Major
-    !if "${QT5}" == "True"
-      IntCmpU $R0 6 0 _os_check_fail _os_check_pass
-      GetWinVer $R1 Minor
-      IntCmpU $R1 2 _os_check_pass _os_check_fail _os_check_pass
-    !else
-      IntCmpU $R0 10 0 _os_check_fail _os_check_pass
-      GetWinVer $R1 Build
-      ${If} $R1 >= 22000 ; Windows 11 21H2
-        Goto _os_check_pass
-      ${ElseIf} $R1 >= 14393 ; Windows 10 1607
-      ${AndIf} ${IsNativeAMD64} ; Windows 10 has no x86_64 emulation on arm64
-        Goto _os_check_pass
-      ${EndIf}
-    !endif
+    IntCmpU $R0 10 0 _os_check_fail _os_check_pass
+    GetWinVer $R1 Build
+    ${If} $R1 >= 22000 ; Windows 11 21H2
+      Goto _os_check_pass
+    ${ElseIf} $R1 >= 14393 ; Windows 10 1607
```

Comment:
unrelated, but note to self: We should update this to 1809 as that's what Qt [requires since at least 6.4](https://doc.qt.io/archives/qt-6.4/supported-platforms.html) ([still in 6.8](https://doc.qt.io/qt-6.8/supported-platforms.html)).

@bitraid How do I find the right value to use for this?

Reply:
Yes this was experimentally verified! https://github.com/qutebrowser/qutebrowser/pull/7315#issuecomment-1675089295
Also a value for build 1809 is further up in that PR discussion.
FWIW I'm still able to install the latest nightlies in my windows VM, which is somehow still build 1803/4. I'm sure it wouldn't be too hard to upgrade it if I had to though.

Reply:
Oh, it does have build numbers on this page actually, in the headers of the collapsible bits at the bottom: https://learn.microsoft.com/en-us/windows/release-health/release-information

Reply:
> Oh, it does have build numbers on this page actually, in the headers of the collapsible bits at the bottom: https://learn.microsoft.com/en-us/windows/release-health/release-information

Yes, of course! I thought he meant how to find which actual Windows build works for the latest QT version.

Reply:
If the Qt docs say "1809 or later" then I'd propose we make the installer check for that. Even if it happens to work on older versions, it might be that subtle things don't?

I'm surprised they don't have a minimum version mentioned there for Windows 11.

Reply:
> I'm surprised they don't have a minimum version mentioned there for Windows 11.

I'm sure it works for every Windows 11 build (Windows 10 1809 is from 2018 - first release build of Windows 11 is from 2021).